src/Symfony/Component/Form/Extension/Core/Type/BaseType.php

```diff
@@ -117,6 +117,7 @@ public function configureOptions(OptionsResolver $resolver)
             'attr' => array(),
             'translation_domain' => null,
             'auto_initialize' => true,
+            'force_submit' => false,
```

Comment:
Does it make sense to support this for buttons? Otherwise, this should be added to `FormType` instead.